### ISSUE TITLE
Add New Class For Sprite Sheet

### ISSUE THREAD

**Issue by theilman7** [baltimorecounty/baltimorecountymd.gov-assets#255]
The CE wants to add an Instagram icon to his section on the home page and thus will require an update to the sprite sheet and CSS. In this case, I will just need a new class added to homepage.min.css. I have attached the CSS string I would like added below. I have also attached a link to preview what this will look like, mocked up in dev (see County Executive section social media icons, right sidebar).

**Dev Preview:** 
https://dev.baltimorecountymd.gov/index.html

**Styles**

```css
.social-media-list-gold .social-media-list-item-instagram {
    background-position: -189px -74px;
}

.social-media-list-gold .social-media-list-item-instagram:hover {
    background-position: -189px -111px;
}
```

**Comment by martypowell** [comment]
this is in progress. @danfox01 sinec this is such a small change, i'd like to do it in the form of a hotfix and push asap.

**Comment by danfox01** [comment]
Released as HF 1.18.3

https://github.com/baltimorecounty/baltimorecountymd.gov-assets/releases/tag/1.18.3
